vaccine_feed_ingest/runners/ca/metrolink/normalize.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+
+from vaccine_feed_ingest_schema import schema  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+_source_name = "ca:metrolink"
+
+
+def _get_id(site: dict) -> str:
+    name = _get_name(site)
+    city = _get_city(site)
+    return f"{name}:{city}"
```

Comment:
Let's avoid spaces in the `id` field! There was some good discussion on potential alternatives in #223 - could we try one of those techniques (or simply substitute the spaces for a `-`)? 

Note: we have not consistently enforced no-spaces-in-ids thusfar, but it's come up a few times, so I'm trying to make this a bit more future-proof.

Reply:
Done, with hyphens. Any other characters we should strip?